e2e/rust/src/main.rs

```diff
@@ -1,20 +1,31 @@
-use candid::candid_method;
-use ic_cdk_macros::update;
+use std::str::FromStr;
 
-mod declarations;
+use candid::{candid_method, Principal};
+use ic_cdk_macros::update;
 
-use declarations::EVM_RPC_STAGING::{
-    BlockTag, EthMainnetService, GetBlockByNumberResult, MultiGetBlockByNumberResult,
-    ProviderError, RpcError, RpcService, RpcServices, EVM_RPC_STAGING as evm_rpc,
+use evm_rpc_types::{
+    Block, BlockTag, EthMainnetService, Hex32, MultiRpcResult, ProviderError, RpcError, RpcService,
+    RpcServices,
 };
 
 fn main() {}
 
+#[cfg(target_arch = "wasm32")]
+const CANISTER_ID: Option<&str> = Some(std::env!(
+    "CANISTER_ID_EVM_RPC_STAGING",
+    "Unspecified canister ID environment variable"
+));
+#[cfg(not(target_arch = "wasm32"))]
+const CANISTER_ID: Option<&str> = None;
```

Comment:
This removes the environment variable error in the IDE and when running `cargo test`, although maybe there is a nicer way to do this.